providers/common/sql/src/airflow/providers/common/sql/hooks/sql.py

```diff
@@ -804,10 +806,15 @@ def _make_common_data_structure(self, result: T | Sequence[T]) -> tuple | list[t
             return cast("list[tuple]", result)
         return cast("tuple", result)
 
+    def _truncate_text(self,text):
```

Comment:
```suggestion
    def _truncate_text(self,text:str) -> str:
```

lets add type hints for the parameters and return types